schematic/models/validate_attribute.py

```diff
@@ -1291,94 +1357,88 @@ def _merge_format_invalid_rows_values(
+          output and rule type
 
+        Args:
+            val_rule (str): Validation Rule
+            source_attribute (str): Source manifest column name
+            validation_output (SetValidationOutput): The validation putput to parse into
+              error and warning messages
         Returns:
-            errors, list[str]: list of errors to raise, as appropriate, if values in current manifest do
-            not pass relevant cross mannifest validation across the target manifest(s)
-            warnings, list[str]: list of warnings to raise, as appropriate, if values in current manifest do
-            not pass relevant cross mannifest validation across the target manifest(s)
+            tuple[list[str], list[str]]: A list of error message and a list of warning messages
         """
-        errors, warnings = [], []
 
-        (
-            missing_manifest_log,
-            present_manifest_log,
-            repeat_manifest_log,
-        ) = set_validation_store
-
-        # Process error handling by rule
-        if "matchAtLeastOne" in val_rule and len(present_manifest_log) < 1:
-            (
-                invalid_rows,
-                invalid_entries,
-                manifest_ids,
-            ) = self._parse_validation_log(validation_log=missing_manifest_log)
-            errors, warnings = self._get_cross_errors_warnings(
-                val_rule=val_rule,
-                row_num=invalid_rows,
-                attribute_name=source_attribute,
-                invalid_entry=invalid_entries,
-                manifest_id=manifest_ids,
-            )
+        errors: list[str] = []
+        warnings: list[str] = []
+        msg: Optional[str] = None
 
-        elif "matchExactlyOne" in val_rule and len(present_manifest_log) != 1:
-            errors, warnings = self._get_cross_errors_warnings(
-                val_rule=val_rule,
-                attribute_name=source_attribute,
-                matching_manifests=present_manifest_log,
-            )
+        if (
+            "matchAtLeastOne" in val_rule
+            and len(validation_output.matching_manifests) == 0
+        ):
+            rule = "matchAtLeastOne set"
+            manifest_list = validation_output.target_manifests
+            msg = "Manifest did not match any target manifests"
 
-        elif "matchNone" in val_rule and repeat_manifest_log:
-            (
-                invalid_rows,
-                invalid_entries,
-                manifest_ids,
-            ) = self._parse_validation_log(validation_log=repeat_manifest_log)
-            errors, warnings = self._get_cross_errors_warnings(
-                val_rule=val_rule,
-                row_num=invalid_rows,
-                attribute_name=source_attribute,
-                invalid_entry=invalid_entries,
-                manifest_id=manifest_ids,
+        if (
+            "matchExactlyOne" in val_rule
```

Comment:
Is there any value to extracting these out to a constants file?

Reply:
I moved these to a literal and created a new method to parse out the various parts which creates a new dataclass called 
`ParsedCrossValidationRule`